addons/medical_vitals/functions/fnc_handleUnitVitals.sqf

```diff
@@ -113,6 +113,7 @@ switch (true) do {
         TRACE_3("BloodVolume Fatal",_unit,BLOOD_VOLUME_FATAL,_bloodVolume);
```

Comment:
Did we want to remove the bleedout death condition as part of these changes?

Not sure we ever reached a decision on that